src/Organism/Sponsorship/index.js

```diff
@@ -0,0 +1,153 @@
+import React, { forwardRef } from 'react';
+import useSwr from 'swr';
+import List, { Item } from 'topo/Atom/List';
+import Box from 'topo/Atom/Box';
+import Divider from 'topo/Atom/Divider';
+import { Flex, Icon } from "@chakra-ui/core"
+import Skelly from 'topo/Atom/Skelly';
+import Text, { Heading, Link, CopyText } from 'topo/Atom/Text';
+import Content from 'topo/Molecule/Content';
+import { childrenOfType, makePureBox } from 'topo/_utils';
+import { useString, apiFetch } from 'topo/utils';
+
+// GraphQL Query, pulls data from the sponsorship programs area
+const query = `{
+  cms {
+    programs {
```

Comment:
This query is going to pull all programs, ordered by when they were last edited, and then you're only using the first one. That is unpredictable and won't let us reuse it for different programs anyway.

The SponsorTable component should be updated to take a program webname, and you can rewrite the query to use it:

```gql
query SponsorsQuery($webname: String!) {
 cms {
    programs (where: { webname: $webname }, limit: 1) {
      items {
        sponsorPerks
      }
    }
  }
}
```

Reply:
The latest commit has changed this, let me know if anything else needs to be changed!